tests/debug.rs

```diff
@@ -1820,6 +1820,81 @@ mod generic {
         }
     }
 
+    mod raw {
+        #[cfg(not(feature = "std"))]
+        use alloc::format;
+
+        use derive_more::Debug;
+
+        #[derive(Debug)]
+        struct StructOne<T> {
+            r#thing: T,
+        }
+
+        #[derive(Debug)]
+        enum Enum<T> {
+            One { r#thing: T },
+        }
+
+        #[test]
+        fn assert() {
+            assert_eq!(
+                format!("{:?}", StructOne::<u8> { r#thing: 8 }),
+                "StructOne { r#thing: 8 }"
```

Comment:
shouldn't this be:

```suggestion
                "StructOne { thing: 8 }"
```

i.e. what does std do here?

Reply:
@JelteF nice catch 👍

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=9e4490c4d098257a889a6c70d4f90808